metals/src/main/scala/scala/meta/internal/parsing/Trees.scala

```diff
@@ -32,6 +33,7 @@ final class Trees(
 )(implicit reports: ReportContext) {
 
   private val trees = TrieMap.empty[AbsolutePath, Tree]
+  private val tokens = TrieMap.empty[AbsolutePath, Tokens]
```

Comment:
is this used?

Reply:
ach, that was from another issue, just forgot to remove it.